README.md

```diff
@@ -54,6 +54,42 @@ This code snippet will replace the HTML node `.mod_my_module` when clicking on `
 
 </script>
 ```
+### Ajax Calendar
+Add this to the `cal_default.html5` template and add a class like here `.inn_call_ajax` to the next and previous month links. In this example we have an article with the id `calendar` and the contao calendar as element.
+
+```html
+<script>
+    $("#calendar a.inn_call_ajax").click(function (event) {
+        var element;
+        //Get url of next Month
+        var $url = window.location.origin + '/' + $(this).attr('href');
```

Comment:
I am not sure if an absolute link is required, but it is ok :)

Reply:
> Thank you very much! Can you please accept these suggestions? I changed the selector, so that the code is not dependent on a css id.

Welcome :) Thank you for your suggestions! :) have a nice day